packages/button/README.md

```diff
@@ -169,3 +169,43 @@ const { getButton, isDisabled } = getTestUtils();
 | ------------ | ------------------------------------- | ------------------- |
 | `getButton`  | Returns the input node                | `HTMLButtonElement` |
 | `isDisabled` | Returns whether the input is disabled | `boolean`           |
+
+# Loading
```

Comment:
nit: I feel like this section title could be a bit clearer (that it's referring to how to use the loading props, and not loading the button package).